src/test/e2e/PageObjects/CaseReferenceNumberSearch.page.ts

```diff
@@ -0,0 +1,34 @@
+import {CaseReferenceNumberSearchResultsPage} from './CaseReferenceNumberSearchResults.page';
+
+const helpers = require('../Helpers/Selectors');
+
+export class CaseReferenceNumberSearchPage {
+
+  async getPageTitle(): Promise<string> {
+    $(helpers.SearchTitle).catch(() => {
+      console.log(`${helpers.SearchTitle} not found`);
+    });
+
+    return $(helpers.SearchTitle).getText();
+  }
+
+  async enterText(text: string): Promise<void> {
+    $(helpers.SearchInput).catch(() => {
+      console.log(`${helpers.SearchInput} not found`);
+    });
+
+    const searchInput = await $(helpers.SearchInput);
+    await searchInput.addValue(text);
+    await browser.keys('Escape');
```

Comment:
probs can get rid of this as you have turned autocomplete off

Reply:
removed.